### ISSUE TITLE
TSQR requires Kokkos::Serial enabled

### ISSUE THREAD

**Issue by etphipp** [trilinos/Trilinos#2668]
```
[ 73%] [ 73%] Building CXX object packages/tpetra/tsqr/src/CMakeFiles/kokkostsqr.dir/Tsqr_SeqTest.cpp.o
Building CXX object packages/tpetra/tsqr/src/CMakeFiles/kokkostsqr.dir/Tsqr_CombineTest.cpp.o
In file included from /home/etphipp/project/atdm/sparc/Trilinos/Trilinos/packages/tpetra/tsqr/src/Tsqr_Combine.hpp:50:0,
                 from /home/etphipp/project/atdm/sparc/Trilinos/Trilinos/packages/tpetra/tsqr/src/Tsqr_CombineTest.cpp:48:
/home/etphipp/project/atdm/sparc/Trilinos/Trilinos/packages/tpetra/tsqr/src/Tsqr_CombineNative.hpp:196:69: error: ‘Serial’ is not a member of ‘Kokkos’
          const Kokkos::View<const scalar_type*, Kokkos::LayoutLeft, Kokkos::Serial>& x,
```

Is this intended?

@trilinos/tpetra

**Comment by mhoemmen** [comment]
@etphipp Argh OK I'll fix this, sorry!

**Comment by mhoemmen** [comment]
@etphipp PR #2668 fixes this; please review and approve if OK.  Thanks!

**Comment by mhoemmen** [comment]
Fixed in develop via PR #2668; thanks!
